src/Alert.js

```diff
@@ -1,22 +1,37 @@
 import "./Alert.css";
+import {useEffect} from "react";
 
 function Alert(props) {
+    let textInput = null;
+    useEffect(()=> {
+        textInput.focus();
+    });
     return (
         <div className={"backdrop"} onClick={(e) => props.onClose("", false)}>
-            <div className="modal">
+            <div className="modal" aria-modal="true">
                 {props.children}
                 <div className="alert-buttons">
-                    <button className={"alert-button alert-cancel"} type={"button"}
-                            onClick={(e) => props.onClose("", false)}>
+                    <div  id="first">
+                    <button tabIndex={0}
+                            className={"alert-button alert-cancel"}
+                            type={"button"}
+                            aria-label={"cancel button for ".concat( (props.taskToDeleteParams[1] ? "delete all completed tasks" : "delete task"))}
+                            onClick={(e) => props.onClose("", false)}
+                            ref={(button) => { textInput = button; }}>
                         Cancel
                     </button>
-                    <button className={"alert-button alert-ok"} type={"button"}
+                    </div>
+                    <div id="second">
+                    <button tabIndex={0}
+                            className={"alert-button alert-ok"} type={"button"}
+                            aria-label={"OK button for ".concat( (props.taskToDeleteParams[1] ? "delete all completed tasks" : "delete task"))}
```

Comment:
I might name this "Confirm delete all completed tasks".  No need to provide "button" because Voiceover will already provide that info.